service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatusService.java

```diff
@@ -281,6 +281,26 @@ public void updateEhrExtractStatusAcknowledgement(SendAcknowledgementTaskDefinit
         LOGGER.info("Database updated for sending application acknowledgement");
     }
 
+    public void updateEhrExtractStatusCommon(SendEhrCommonTaskDefinition taskDefinition, String messageId) {
+        Query query = createQueryForConversationId(taskDefinition.getConversationId());
+
+        var commonSentAt = GPC_DOCUMENTS + DOT + taskDefinition.getDocumentPosition() + DOT + "sentToMhs" + DOT + SENT_AT;
+        var commonTaskId = GPC_DOCUMENTS + DOT + taskDefinition.getDocumentPosition() + DOT + "sentToMhs" + DOT + TASK_ID;
+        var commonMessageId = GPC_DOCUMENTS + DOT + taskDefinition.getDocumentPosition() + DOT + "sentToMhs" + DOT + MESSAGE_ID;
```

Comment:
The story says only about two values, sentAt and taskId. Should the description be updated?

Reply:
I decide to add this message-id, its the id of the message sent to mhs so thought it would be useful here. I will update ac

Reply:
yes, it's good idea. I also think it will be useful.